syntax = "proto3";

package datamodel;

message Transaction {
  string transaction_id = 1;
  int32 user_id = 2;
  double amount = 3;
  int64 timestamp = 4;
  string status = 5;
}